test/integration/auth.spec.ts

```diff
@@ -307,8 +311,291 @@ describe('admin.auth', () => {
       admin.auth().deleteUser(uidFromCreateUserWithoutUid),
     ]).should.eventually.be.fulfilled;
   });
+
+  describe('importUsers()', () => {
+    const randomUid = 'import_' + generateRandomString(20).toLowerCase();
+    let rawPassword;
+    let importOptions;
+    let importUserRecord;
+    let rawSalt;
+    let hashKey;
+
+    afterEach(() => {
+      return safeDelete(randomUid);
+    });
+
+    it('successfully imports users with HMAC_SHA256 hashed passwords', () => {
```

Comment:
Is it possible to organize these tests by creating an array of test fixtures? For example:

```
interface UserImportTest {
  name string
  importOptions UserImportOptions
  passwordHash string
}

const fixtures: Array<UserImportTest> = [
  {
     name: 'hmac',
     importOptions: {...},
     passwordHash: crypto.createHmac(...),
  },
  ...
]; 

fixtures.forEach((fixture) => {
   it(...)
});
```